exe/ReferenceDoc.hs

```diff
@@ -105,7 +110,7 @@ main = do
 
     allFns =
          basics ++ maths ++ evalFns ++ envStuff ++ seqs ++ lists ++ vecs
-      ++ dicts ++ structs ++ refs ++ docs ++ io ++ lens ++ crypto ++ chainTools
+      ++ dicts ++ structs ++ refs ++ docs ++ io ++ lens ++ validation ++ crypto ++ chainTools
```

Comment:
Maintaining this is very quickly going to get annoying. Should it perhaps not be automatically generated from the file order, with perhaps some way of documenting a module intro (or sections)?

Reply:
I take this back. The tooling around this is already super nice! Hats off!